### DISCUSSION TITLE
Error Install Prophet 1.1.5

### DISCUSSION THREAD

**Discussion by wildaan24** [teamalgoritma/community#919] [Q&A] [answered]
Dear Algoritma team, 

Sebelumnya sorry saya menemukan kendala ketika melakukan running code dibagian instal prophet.

![Prophet1](https://github.com/user-attachments/assets/b116be11-c729-4404-9fc9-b68d354a956a)
![Prophet2](https://github.com/user-attachments/assets/e6db3f9c-6f61-423a-b882-8882c1af6eda)

ini kondisi ketika file requirements untuk prophetnya sudah versi 1.1.5

Terus, saya coba melakukan downgrade ke versi 1.1.1 tetapi hasilnya seperti ini 

![Prophet3](https://github.com/user-attachments/assets/6af5f332-4a77-4d9d-9abb-143b8f43e213)
![prophet4](https://github.com/user-attachments/assets/24a36b75-dd6e-4741-ab87-bfb4308cb2a8)

Mohon pencerahannya, terimakasih.

**Comment by adizaen** [comment, marked as answer]
Dear Pak @wildaan24 

Terima kasih Pak atas pertanyaannya. Setelah kami telusuri, bisa dicoba untuk melakukan instalasi ulang library Prophet dengan menambahkan keyword "-m" pada perintah install. Berikut kode install nya:

`python -m pip install prophet==1.1.1`

Setelah berhasil, silakan Bapak coba untuk jalankan kode ipynb nya. Jika masih menjumpai error, coba lakukan downgrade juga untuk library Holidays ke versi 0.18 dengan kode install sebagai berikut:

`python -m pip install holidays==0.18.0`

Semoga berhasil Pak Wildan,

**Best regards,**
Team Algoritma

**Reply by adizaen** [reply]
Dear Pak @wildaan24 

Menindaklanjuti balasan sebelumnya, apakah Bapak berkenan untuk melaporkan versi Python yang Bapak gunakan?

**Comment by adizaen** [comment]
Dear @wildaan24 

Terima kasih telah bertanya melalui Github discussions Algoritma. Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

**Best regards,**
Team Algoritma
